APE_specutils.rst

```diff
@@ -0,0 +1,147 @@
+
+
+Spectroscopic Coordinating Committee
+++++++++++++++++++++++++++++++++++++
+This APE formally names a committee for coordinating this effort (although this committee has been operating in an informal manner well prior to this APE).  Patterned after the Astropy Coordinating Committee, the Spectroscopic Coordination Committee will be charged with overseeing the effort described below, as well as being a central point of communication for Astropy spectroscopy efforts. One member of the committee will be a liaison member who is also a member of the Astropy Coordination Committee.  
+
+The path forward
+++++++++++++++++
+This APE proposes a coordinated suite of packages for doing astronomical spectroscopy in Python.  There are two tiers to this coordinated plan: this APE is concerned primarily with the first tier, which is for development of “standard” tools (i.e., those that have relatively little debate about implementation, or are historically well-used). The intent is for this tier to develop a baseline set of tools and interfaces.  This tier will be maintained by the Astropy community specifically to provide these base tools and interfaces that will allow all the packages to work with each other when possible.  
+
+The packages will be developed separately but are connected by a common interface object: `Spectrum1D`. This object will be in a common package - for the near-term, the common package will be `specutils`, but once the interface has stabilized, these interface classes may be moved to the `astropy` core package. Packages should use this common `Spectrum1D` object (described in more detail below) for representing spectra, although they are welcome to sub-class it to add package-specific functionality.  Also note that, in the future, more interface objects might be called for - e.g. `Spectrum2D` or `SpectralCube` or similar.  In previous discussions at Astropy coordination meetings, there has also been suggestions to eventually create domain-specific subclasses like `RadioSpectrum1D` or `XRaySpectrum1D` (with e.g. different default units appropriate for that field).  The intent is for `Spectrum1D` to provide a starting point for such efforts, which will then develop in a similar manner as they are called for by the community.
+
+The foundational tier of packages will consist of three packages: 
+
+* `specutils <https://github.com/astropy/specutils)>`_.  This will provide the basic interface classes, as well as *basic* analysis of spectroscopic data. “Basic” here simply means highly standard pieces like measuring equivalent widths, fluxes, Gaussian/Lorentzian/Voigt fitting, RV corrections, etc.
```

Comment:
I'm not sure if everything in specutils should be moved to the core, but I would agree that the Spectrum1D object should move into the core at the very least (and probably be done sooner rather than later)